cfy_manager/components/usage_collector/scripts/script_utils.py

```diff
@@ -78,8 +92,16 @@ def collect_metadata(data):
     }
 
 
-def send_data(data, url):
+def send_data(data, url, interval_type):
     # for some reason. multi hierarchy dict doesn't pass well to the end point
+    with open(USAGE_CONFIG_PATH) as usage_config_file:
+        usage_config = json.load(usage_config_file)
+    if interval_type == HOURS_INTERVAL:
+        usage_config[HOURLY_TIMESTAMP] = time.time()
```

Comment:
we could do something like, check if `hasattr(time, 'monotonic')` and then use that, otherwise use `time.time()`

`time.monotonic()` is a timer that doesnt go backwards after you change the system time, but it's only available in py3.3+

or just ignore this for now and somehow remember to do it later after py3 update

Reply:
Nice, seems like a great solution for the case that the system time would change. I think I'll keep it for after the py3 update.